lib/ask_export.rb

```diff
@@ -0,0 +1,40 @@
+require "active_support/core_ext"
+Time.zone = "Europe/London"
+
+require_relative "ask_export/csv_builder"
+require_relative "ask_export/csv_splitter"
+require_relative "ask_export/partner_notifier"
+require_relative "ask_export/runner"
+require_relative "ask_export/survey_response_fetcher"
+
+module AskExport
+  CONFIG = {
+    draft: {
+      survey_id: 741027,
+      region_field_id: 11348121,
+      question_field_id: 11348119,
+      question_format_field_id: 11348124,
+      name_field_id: 11348120,
+      email_field_id: 11348122,
+      phone_field_id: 11348123,
+    },
+    live: {
+      survey_id: 736162,
+      region_field_id: 11312915,
+      question_field_id: 11288904,
+      question_format_field_id: 11324295,
+      name_field_id: 11289065,
+      email_field_id: 11289069,
+      phone_field_id: 11312922,
+    },
+  }.freeze
+
+  def self.config(item, default = :draft)
```

Comment:
The `default` aspect of this confused me for quite a while (particularly when reading [this](https://github.com/alphagov/govuk-ask-a-question/pull/3/files#diff-3204074c98b93b7a28d931bda149b580R78)) until I saw [this](https://github.com/alphagov/govuk-ask-a-question/pull/3/files#diff-6aee983c514b10fe42f515ef1cb0a2a3R50). I understand the need for a different default when running the manual task, as `:draft` would be surprising. Could we support that need by `||=`ing the environment variable in the rake task, instead?

Reply:
Yeah, that's a good idea.

Reply:
I've added that in.